docs/installation/bootstrap.rst

```diff
@@ -187,16 +205,13 @@ Check if all :term:`Pods <Pod>` on the Bootstrap node are in the
    metalk8s-monitoring   prometheus-k8s-1                                 3/3       Running   1          3m        10.233.132.78   bootstrap   <none>
    metalk8s-monitoring   prometheus-operator-64477d4bff-xxjw2             1/1       Running   0          4m        10.233.132.68   bootstrap   <none>
 
-Check that you can access the MetalK8s GUI, following
-:ref:`this procedure <quickstart-services-admin-ui>`.
-
-Troubleshooting
-^^^^^^^^^^^^^^^
+- Check and verify that you can access the MetalK8s GUI after the
+  :ref:`installation <Bootstrap installation>` is completed by following
+  :ref:`this procedure <quickstart-services-admin-ui>`.
 
-.. todo::
+- At this stage, the MetalK8s GUI should be up and ready for you to
```

Comment:
even if prom and am PV have not been provisioned?

Reply:
fixed